static/app/components/charts/simpleTableChart.tsx

```diff
@@ -59,7 +68,42 @@ function SimpleTableChart({
       const fieldRenderer =
         getCustomFieldRenderer?.(column.key, tableMeta) ??
         getFieldRenderer(column.key, tableMeta);
-      const rendered = fieldRenderer(row, {organization, location});
+      let rendered = fieldRenderer(row, {organization, location});
+      if (column.key === 'id') {
+        const eventSlug = generateEventSlug(row);
+
+        const target = eventDetailsRouteWithEventView({
+          orgSlug: organization.slug,
+          eventSlug,
+          eventView,
+        });
+
+        rendered = (
+          <Tooltip title={t('View Event')}>
+            <Link data-test-id="view-event" to={target}>
+              {rendered}
+            </Link>
+          </Tooltip>
+        );
+      } else if (column.key === 'trace') {
+        const dateSelection = eventView.normalizeDateSelection(location);
+        if (row.trace) {
+          const target = getTraceDetailsUrl(
+            organization,
+            String(row.trace),
+            dateSelection,
+            {}
+          );
+
+          rendered = (
+            <Tooltip title={t('View Trace')}>
+              <Link data-test-id="view-trace" to={target}>
+                {rendered}
+              </Link>
+            </Tooltip>
+          );
+        }
+      }
```

Comment:
is there a reason we can't add field renderers for these fields in `fieldRenderers.tsx` so that `getFieldRenderer` returns the rendered component as we want? It looks like we're using the event view to create the target urls, but can they be constructed without the event view (and just using the event id/trace id and location params?)

It would be nice to keep this logic contained in that file so we're not doing one-off modifications here!

Reply:
So I agree with you for sure, and an earlier iteration of this change attempted that,
But unfortunately it has to be one-off cause performance uses the same fieldRenderer, but links differently. I _think_ there's an opportunity to refactor fieldRenderer here so that we can do it differently for different parts of the app but I want to keep this change contained to just getting this missing feature in for now instead of doing a refactor

Reply:
@wmak @shruthilayaj You can have a higher level wrapper around `fieldRenderer()` that's specific for the Dashboards product; as opposed to treating it as a one-off change here.

Reply:
Looks like what I suggested can be delegated to `getCustomFieldRenderer()`.

Reply:
yeah getCustomFieldRenderer() might work here, ok I'll make a jira ticket for this and take a look into refactoring

Reply:
also, aside (will help with the refactor), the target urls can be constructed without the event view too right?